WindowsServerDocs/manage/windows-admin-center/configure/use-powershell.md

```diff
@@ -14,6 +14,8 @@ If you have a large organization with multiple Windows Admin Center servers, you
 
 ## Use PowerShell to import or export your connections (with tags)
 
+[!NOTE] With version 2103 there is a known bug with the included modules requiring modification to the header parameters shown in this [Gist](https://gist.github.com/Snozzberries/3ed0a64f691b262d2b5edcbfdf4d2a37).
```

Comment:
Good start, but we also need the MarkDown formatting to make it work properly as a Note blob:
```suggestion
> [!NOTE]
> With version 2103, there is a known bug with the included modules, requiring modification to the header parameters shown in this [Gist](https://gist.github.com/Snozzberries/3ed0a64f691b262d2b5edcbfdf4d2a37).
```

Reply:
@Snozzberries : Will you be able to return to this PR, to review & merge the suggested improvement?

Reply:
@illfated Sorry thought I had accepted your suggestion. New commit added.